pkg/mount/mountutil.go

```diff
@@ -112,6 +115,77 @@ func getDiskDevPath(partition string) (string, error) {
 	return "/dev/" + parentDisk, nil
 }
 
+//	getSoftLinkForPartition returns path to /sys/class/block/$partition
+//	if the path does not exist and the partition is "root"
+//	then the root partition is detected from /proc/cmdline
+func getSoftLinkForPartition(partition string) (string, error) {
+	softlink := getLinkForPartition(partition)
+
+	if _, err := os.Stat(softlink); os.IsNotExist(err) && partition == "root" {
+		partition, err := getRootPartition()
+		if err != nil {
+			return "", err
+		}
+		softlink = getLinkForPartition(partition)
+	}
+	return softlink, nil
+}
+
+//	getLinkForPartition returns path to sys block path
+func getLinkForPartition(partition string) string {
+	// dev path be like /dev/sda4 we need to remove /dev/ from this string to get sys block path.
+	return "/sys/class/block/" + partition
+}
+
+//	getRootPartition resolves link /dev/root using /proc/cmdline
+func getRootPartition() (string, error) {
+	file, err := os.Open("/proc/cmdline")
```

Comment:
This will take `/proc` from the container right? We have the host `procfs` mounted at `/host/proc`. 
Should we use `/host/proc/cmdline` here instead?

Reply:
yes, that's true. I checked the file inside of a container and it had the same content.
I can use /host/proc/cmdline and if it does not exists; I can check /proc/cmdline as a fallback (if not running in a container).
Is there any use-case when NDM would not run inside of a container?

Reply:
> I can use /host/proc/cmdline and if it does not exists; I can check /proc/cmdline as a fallback (if not running in a container).

That would be great.



> Is there any use-case when NDM would not run inside of a container?

Nothing that I can think of now. cc: @kmova

Reply:
support for /host/proc/cmdline and using /proc/cmdline as fallback => implemented